packages/cli/src/commands/dev.ts

```diff
@@ -5,6 +5,14 @@ import * as ora from 'ora';
 import * as open from 'open';
 import * as chalk from 'chalk';
 import { SERVER_HOST } from '../constants';
+import { v4 as uuidv4 } from 'uuid';
+import { AnalyticService, ConfigService } from '../services';
+
+const analytics = new AnalyticService();
+const config = new ConfigService();
+if (process.env.NODE_ENV === 'development') {
```

Comment:
NODE_ENV won't be set in the separate TTY the developers use to run this command. When do we want to trigger the config clearance?